python-avd/pyavd/_cv/client/tag.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 from typing import TYPE_CHECKING, Literal
 
+from pyavd._cv.api.arista.time import TimeBounds
+
 from ..api.arista.tag.v2 import (
```

Comment:
```suggestion
from pyavd._cv.api.arista.tag.v2 import (
```

Reply:
yeah this one is tough as this wont be consistent with other files in 4.10 as we moved to absolute path later in devel but lets be consistent at file level